python/lsst/summit/utils/consdbClient.py

```diff
@@ -0,0 +1,412 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import os
+from typing import Any
+from urllib.parse import quote
+
+import requests
+from astropy.table import Table  # type: ignore
+
+__all__ = ["ConsDbClient"]
+
+
+logger = logging.getLogger(__name__)
+
+
+def _urljoin(*args: str) -> str:
+    """Join parts of a URL with slashes.
+
+    Does not do any quoting.  Mostly to remove a level of list-making.
+
+    Parameters
+    ----------
+    *args: `str`
+        Each parameter is a URL part.
+
+    Returns
+    -------
+    url: `str`
```

Comment:
It might not really matter, but I think we generally do
```
paramater : `type`
```
 _i.e._ with a space before the colon.